packages/mui-codemod/src/v6.0.0/list-item-button-prop/test-cases/theme.expected.js

```diff
@@ -0,0 +1,21 @@
+fn({
```

Comment:
@DiegoAndai , I am not sure about these transformations honestly, as there could be different interpretation on what was the intention. I did what felt most intuitive to me, but I am honestly not happy with it. I think I would rather not handle this and let people do what they think should be best.

Reply:
If I understand correctly, the logic is: If `MuiListItem` is present and has `button` as default prop, then remove `button` and create a `MuiListItemButton` entry copying the other default prop values, right? That is the most likely migration to me.

What other interpretations of the intention are you thinking about?